apps/docs-snippets2/src/introduction/connecting-to-localnode.test.ts

```diff
@@ -0,0 +1,11 @@
+import { main } from './connecting-to-testnet.wrapped';
```

Comment:
In tests, wrapped (generated) snippets should be imported instead of the original ones.